toplevel/coqinit.ml

```diff
@@ -59,13 +59,23 @@ let load_rcfile ~time doc sid =
      doc, sid)
 
 (* Recursively puts dir in the LoadPath if -nois was not passed *)
-let add_stdlib_path ~load_init ~unix_path ~coq_root ~with_ml =
-  let add_ml = if with_ml then Mltop.AddRecML else Mltop.AddNoML in
-  Mltop.add_rec_path add_ml ~unix_path ~coq_root ~implicit:load_init
+let build_stdlib_path ~load_init ~unix_path ~coq_path ~with_ml =
```

Comment:
Outdated comment?

Reply:
I think `-nois` is still a supported option [alias of `-noinit`]

Reply:
But it doesn't add anything to the load path, it just builds the entry.

Reply:
Ah OK, will clarify indeed, thanks.

Reply:
well, it says "puts", I'm not sure how to improve it.